changelog/upgrade-notes.md

```diff
@@ -2,6 +2,39 @@
 
 ### Next release
 
+#### Removal of the renderer option for maps
+
+The `renderer` option has been removed from the `Map` constructor.  The purpose of this change is to avoid bundling code in your application that you do not need.  Previously, code for both the Canvas and WebGL renderers was included in all applications - even though most people only use one renderer.  The `Map` constructor now gives you a Canvas (2D) based renderer now.  If you want to try the WebGL renderer, you can import the constructor from `ol/WebGLMap`.
```

Comment:
duplicated word 'now' in 

'The `Map` constructor now gives you a Canvas (2D) based renderer now'

Reply:
Thanks for the catch @marcjansen.  Fixed in c7a10fc7b68c85f9ce26c696dbbed3ea0bd880a5.